stories/alert.stories.js

```diff
@@ -12,117 +10,103 @@ import {
 } from '../src';
 import INTENTS from '../src/Alert/intents';
 
-const ON_CLOSE_OPTIONS = ['', () => alert('Close Me!')]; // eslint-disable-line no-alert, no-undef
+const ON_CLOSE_OPTIONS = [null, () => alert('Close Me!')]; // eslint-disable-line no-alert, no-undef
 
 export default {
-  title: 'Alerts'
+  title: 'Alerts',
+  argTypes: {
+    intent: {
+      name: 'intent',
+      control: {
+        type: 'select',
+        options: INTENTS
+      }
+    },
+    onClose: {
+      name: 'onClose',
+      control: {
+        type: 'select',
+        options: ON_CLOSE_OPTIONS
+      }
+    }
+  }
 };
 
-export const Banner = () => (
+export const Banner = (args) => (
   <Box as="section" p="regular">
     <Heading.H1>Banner Alert</Heading.H1>
 
-    <BannerAlert
-      details={text(
-        'Details',
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.'
-      )}
-      intent={select('Intent', INTENTS, INTENTS[0])}
-      message={text('Message', 'Hooray!')}
-      onClose={select('On Close', ON_CLOSE_OPTIONS)}
-    />
+    <BannerAlert {...args} />
```

Comment:
In the happy case one just spreads `args` onto the component.  But some stories' `args` aren't direct prop-mappings, so in some cases there is a little more code/customization.